app/src/main/cpp/skyline/gpu/interconnect/graphics_context.h

```diff
@@ -786,6 +786,22 @@ namespace skyline::gpu::interconnect {
             }
         }
 
+        BufferView GetSSBOViewFromDescriptor(const ::Shader::StorageBufferDescriptor &descriptor, const std::array<ConstantBuffer, maxwell3d::PipelineStageConstantBufferCount> &constant_buffers) {
+            auto &cbuf{constant_buffers[descriptor.cbuf_index]};
+            IOVA buffer_iova{cbuf.Read<u64>(descriptor.cbuf_offset)};
```

Comment:
camelCase

Reply:
Oh right. I think I got confused by the Yuzu shader compiler code which uses snake_case.